code/modules/hardsuits/_rig.dm

```diff
@@ -121,6 +121,37 @@
 
 	var/sprint_slowdown_modifier = 0					      // Sprinter module modifier.
 
+	//* Storage *//
+	var/list/storage_insertion_whitelist
+	var/list/storage_insertion_blacklist
+	var/list/storage_insertion_allow
+
+	var/storage_max_single_weight_class = WEIGHT_CLASS_NORMAL
+	var/storage_max_combined_weight_class
+	var/storage_max_combined_volume = WEIGHT_VOLUME_NORMAL * 7
+	var/storage_max_items
+
+	var/storage_weight_subtract = 0
+	var/storage_weight_multiply = 1
+
+	var/storage_allow_quick_empty = TRUE
+	var/storage_allow_quick_empty_via_clickdrag = TRUE
+	var/storage_allow_quick_empty_via_attack_self = FALSE
+
+	var/storage_sfx_open = "rustle"
+	var/storage_sfx_insert = "rustle"
+	var/storage_sfx_remove = "rustle"
+
+	var/storage_ui_numerical_mode = FALSE
+
+	/// storage datum path
+	var/storage_datum_path = /datum/object_system/storage
+	/// Cleared after Initialize().
+	/// List of types associated to amounts.
+	var/list/storage_starts_with
+	/// set to prevent us from spawning starts_with
+	var/storage_empty = FALSE
+
```

Comment:
With the amount of vars you need to add to something for this system, we might as well contain it inside another datum...

Reply:
rigsuits update deals with it, and it actually does contain it in another datum

this is a blocker for rigsuits update :^)